packages/nimble-components/.storybook/manager.js

```diff
@@ -3,5 +3,15 @@ import theme from './theme';
 
 addons.setConfig({
     enableShortcuts: false,
+    sidebar: {
+        filters: {
+            patterns: item => {
+                return (
+                    !item.title.startsWith('Internal/')
```

Comment:
> I looked into conditionally hiding the Tests stories when deployed but couldn't think of a good way to do so. I guess we could add a filter based on the page URL being on nimble.ni.dev but that seemed brittle/hacky to me. Open to suggestions!

I think that sounds like a good option to me. Best balance of nimble dev and end user experience. Nimble devs can easily find the latest storybook link if they want to jump to that to see all the latest matrix tests (which would be hosted on chromatic.com) and removes a bunch of noise from the storybook for end users on `nimble.ni.dev`.

Could even consider only hiding all of them only on `nimble.ni.dev` so it's easier to poke around the internal stories if needed from a chromatic run.

Reply:
I think another option would be to create a global where the default value hides all the internal stories: https://storybook.js.org/docs/react/essentials/toolbars-and-globals

I think that we should be able to access the value of globals from manager.js but not 100% certain

Reply:
I went with `window.location.hostname === 'nimble.ni.dev'` after trying it out in dev tools.